lib/install/management/create_test.go

```diff
@@ -90,13 +91,16 @@ func TestMain(t *testing.T) {
 		// The other way around, if we test positive case first, the VCH data and session data are modified, so we are not able to test the negative case
 		conf2, err := validator.Validate(op, input)
 		conf2.ImageStores[0].Host = "http://non-exist"
-		testCreateAppliance(op, validator.Session, conf2, installSettings, true, t)
+		_ = testCreateAppliance(op, validator.Session, conf2, installSettings, true, t)
```

Comment:
Why did you add it?

Reply:
Added since the returned appliance VM can be used in the `testRelocateAppliance` func below.